admission/hooks/validate_subnamespaceanchor_test.go

```diff
@@ -0,0 +1,36 @@
+package hooks
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func fillSubnamespaceAnchor(name string) client.Object {
+	subns := &unstructured.Unstructured{}
+	subns.SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   "hnc.x-k8s.io",
+		Version: "v1alpha2",
+		Kind:    "SubnamespaceAnchor",
+	})
+	subns.SetName(name)
+	subns.SetNamespace("default")
+	subns.UnstructuredContent()["spec"] = map[string]interface{}{}
+	return subns
+}
+
+var _ = Describe("validate SubnamespaceAnchor webhook with ", func() {
```

Comment:
Is there anything word after `with `?

Reply:
No, but there are several other tests using similar sentences.

Reply:
I see.